### DISCUSSION TITLE
No more release since 1.24?

### DISCUSSION THREAD

**Discussion by tlesmann** [dani-garcia/vaultwarden#2467] [Q&A] [answered]
Hi vaultwarden guys,
I'm just wondering about the release strategy. The latest release 1.24 is from Jan. 2022. 
There are many updates in main branch and the web vault i.e. is updated continously. But this doesn't result into any new releases of vaultwarden container in docker hub. Is there a reason for that or should users install the testing branches/tags now?

Thanks for the real cool work and kind regards ...

**Comment by ntimo** [comment]
I also would like to suggest to create a new GitHub release if possible :)

**Comment by BlackDex** [comment, marked as answer]
There have been a lot of changes since the latest release. Some minor issue were only discovered just recently and fixed. So we probably want to have this running on testing for a little while to see if this is all fixed now. While they weren't breaking issues, we would still like it to be as stable as possible.

If you really want to use a later release i suggest to use the `testing` tagged docker image.
